templates/layouts/one-column-overlay.html.twig

```diff
@@ -0,0 +1,24 @@
+{% if 'centered-container' in settings.centered|render and region_attributes.main is iterable %}
+  {% set region_attributes_main = region_attributes.main.addClass('flex-12-of-12') %}
+  {% set region_attributes = region_attributes|merge({'main': region_attributes_main}) %}
+{% endif %}
+
+{% if layout_builder_admin %}
+  <div{{ attributes.addClass('jumpstart-ui--one-column', settings.centered, settings.extra_classes) }}>
+    <div {{ region_attributes.main.addClass('main-region') }}>
+      <div>{{ 'Background Image'|t }}</div>
+      {{ content.main }}
+    </div>
+
+    <div {{ region_attributes.overlay.addClass('overlay-region') }}>
+      <div>{{ 'Overlaying Contents'|t }}</div>
+      {{ content.overlay }}
+    </div>
+  </div>
+{% else %}
+  {% set hero_body = content.overlay %}
+  {% set hero_image = content.main %}
+  {% include "@decanter/components/hero/hero.twig" %}
```

Comment:
Might want to include a data variable and use the `only` keyword to prohibit variable leakage.

Reply:
huh?

Reply:
I just looked and we aren't doing it anywhere else in jumpstart_ui so nevermind but I'll show you what I meant.